pkgs/tools/system/lshw/default.nix

```
@@ -1,40 +1,40 @@
-{ stdenv, lib, fetchurl, fetchpatch
-, withGUI ? false, gtk2, pkg-config, sqlite # compile GUI
+{ stdenv
+, lib
+, fetchFromGitHub
+, hwdata
+, gtk2
+, pkg-config
+, sqlite # compile GUI
+, withGUI ? false
 }:
 
+let
+  # Fix repology.org by not including the prefixed B, otherwise the `pname` attr
+  # gets filled as `lshw-B.XX.XX` in `nix-env --query --available --attr nixpkgs.lshw --meta`
+  # See https://github.com/NixOS/nix/pull/4463 for a definitive fix
+  version = "02.19";
+  actualVersion = "B.02.19";
```

```suggestion
  version = "02.19";
  actualVersion = "B.${version}";
```
we don't want to duplicate this but we also don't need this here.

---

Done.